### PR TITLE
build docker images

### PR DESCRIPTION
**Pull request by ndeloof**
build jenkins/jenkins:lts-alpine-*-casc docker images

### CODE REVIEW

**Comment by ndeloof** [comment]
fyi @carlossg

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+WORKDIR /work
+RUN mvn dependency:go-offline
+COPY . /work
+RUN mvn clean install
+
+# --- to be distributed as jenkins/jenkins:lts-alpine-casc
 FROM jenkins/jenkins:lts-alpine
 
-COPY --chown=jenkins target/configuration-as-code.hpi /usr/share/jenkins/ref/plugins/configuration-as-code.jpi
+COPY --chown=jenkins --from=build /work/target/configuration-as-code.hpi /usr/share/jenkins/ref/plugins/configuration-as-code.jpi
```
Comment (carlossg) [inline Dockerfile]: it is missing all the configuration-as-code dependencies

Comment (ndeloof) [reply Dockerfile]: sole dependency is job-dsl (and it's dependency) which we should consider optional (need to update configurator accordingly)

**Comment by ewelinawilkosz** [comment]
should I merge @ndeloof ?

**Comment by ndeloof** [comment]
please don't, work in progress - would need to make jobdsl an optional dependency, or bundle dependencies

**Comment by ndeloof** [comment]
obsolete

**Pull request closed without merging**